lms/djangoapps/courseware/module_render.py

```diff
@@ -430,7 +439,7 @@ def handle_grade_event(block, event_type, event):  # pylint: disable=unused-argu
         grade = event.get('value')
         max_grade = event.get('max_value')
 
-        field_data_cache.set_score(
+        set_score(
```

Comment:
@cpennington maybe we should remove the `user_id` parameter from event than? This would help avoiding confusion in the future and  simplify the code, instead of adding a new check (and hence, adding more code).